src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -9687,6 +9687,48 @@ void ObjectMgr::LoadGossipMenuItems()
     TC_LOG_INFO("server.loading", ">> Loaded " SZFMTD " gossip_menu_option entries in %u ms", _gossipMenuItemsStore.size(), GetMSTimeDiffToNow(oldMSTime));
 }
 
+void ObjectMgr::LoadGossipMenuFriendshipFactions()
+{
+    uint32 oldMSTime = getMSTime();
+
+    _gossipMenuFriendshipFactionsStore.clear();
+
+    //                                               0       1
+    QueryResult result = WorldDatabase.Query("SELECT MenuID, FactionID FROM gossip_menu_friendship_faction");
+
+    if (!result)
+    {
+        TC_LOG_INFO("server.loading", ">> Loaded 0 gossip_menu_friendship_faction IDs. DB table `gossip_menu_friendship_faction` is empty!");
+        return;
+    }
+
+    do
+    {
+        Field* fields = result->Fetch();
+
+        uint32 MenuID = fields[0].GetUInt32();
```

Comment:
local variables always lowerCamelCase

Reply:
That's what I get for copying the name from the SQL query 😄 Fixed